lib/scripts/users_list.rb

```diff
@@ -26,4 +29,16 @@ def select_all_users
     User.all.select(:id, :formal_name, :username, :email,
```

Comment:
Why is this at the end? Is it for testing to verify the output?

Reply:
Do you mean the method itself? It was here for the same reason from your other comment: tiny methods to make it easier to understand. Or do you mean the `select` here? Or something else?

Reply:
I mean't why is it running `select` for all the users at the end? What is using the returned selection?

Reply:
Ah, I see. `select` here just retrieves the certain fields I need, omitting all the other fields. https://apidock.com/rails/v4.2.7/ActiveRecord/QueryMethods/select